pkg/bindings/containers/commit.go

```diff
@@ -33,7 +33,12 @@ func Commit(ctx context.Context, nameOrID string, options *CommitOptions) (entit
 		return entities.IDResponse{}, err
 	}
 	params.Set("container", nameOrID)
-	response, err := conn.DoRequest(ctx, nil, http.MethodPost, "/commit", params, nil)
+	var requestBody io.Reader
+	if config := params.Get("config"); config != "" {
+		requestBody = strings.NewReader(config)
+		params.Del("config")
+	}
```

Comment:
you can add `schema:"-"` to the type then it will not be added via ToParams()

Reply:
Ah, thanks, I wouldn't have guessed that.